src/Microsoft.Win32.Registry.AccessControl/pkg/Microsoft.Win32.Registry.AccessControl.pkgproj

```diff
@@ -6,11 +6,11 @@
       <SupportedFramework>net461;netcoreapp2.0;$(AllXamarinFrameworks)</SupportedFramework>
     </ProjectReference>
     <ProjectReference Include="..\src\Microsoft.Win32.Registry.AccessControl.csproj" />
-    <NotSupportedOnTargetFramework Include="netcore50">
```

Comment:
Just so I'm clear your goal is now that anytime we would have done this in the past we would now put a GeneratePlatformNotSupported assembly in its place?